Porting/release_managers_guide.pod

```diff
@@ -173,6 +173,15 @@ asked to compare the installed files with a previous install. Save yourself
 some time on release day, and have a (clean) install of the previous
 version ready.
 
+=head3 Email account subscribed to perl5-porters
+
+In order for your release announcement email to be delivered to the
+perl5-porters distribution list, the email address that you intend to
+send from must be subscribed to the list.
+
+Instructions for subscribing can be found here:
+L<List: perl5-porters|https://lists.perl.org/list/perl5-porters.html>
+
```

Comment:
Did you get a bounce when trying to send?

AFAIK this is untrue, messages are delayed for moderation if you haven't posted to the list recently.

Reply:
I had to use a new email account to send the announcement. I hadn't subscribed it to the list before sending the announcement, which neither bounced nor got delivered. 

I asked on IRC if it was stuck in moderation, but hadn't heard anything after multiple hours. Someone suggested that the problem was that the account wasn't subscribed. I subscribed and retried sending the announcement, and it was then promptly delivered.